<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node pkg="rovio" type="rovio_node" name="rovio" output="screen">
  <param name="filter_config" value="$(find rovio)/cfg/rovio_intel1.info"/>
  <param name="camera0_config" value="$(find rovio)/cfg/intel1_cam.yaml"/>
  <remap from="cam0/image_raw" to="/fisheye/fisheye"/>
  <remap from="imu0" to="/imu/raw"/>
  </node>
</launch>
